### ISSUE TITLE
`deadline_test` command not working

### ISSUE THREAD

**Issue by locnnil** [canonical/rt-tests-snap#8]
The `deadline_test` command isn't working, interface access is missing:

AppArmor log:
```log
= AppArmor =
Time: Jun 13 17:08:36
Log: apparmor="DENIED" operation="open" class="file" profile="snap.rt-tests.deadlinetest" name="/proc/1335293/mounts" pid=1335293 comm="deadline_test" requested_mask="r" denied_mask="r" fsuid=0 ouid=0
File: /proc/1335293/mounts (read)
Suggestions:
* adjust program to not access '@{PROC}/@{pid}/mounts'
* add one of 'mount-control, mount-observe, network-control, steam-support' to 'plugs'
```

**Comment by farshidtz** [comment]
Closed via #18

**Comment by farshidtz** [comment]
Reopened as this wasn't merged into main, but rather a fix branch.
